src/main/java/com/minecolonies/structures/lib/RenderUtil.java

```diff
@@ -19,6 +19,8 @@ private RenderUtil()
 
     public static void applyRotationToYAxis(@NotNull final Rotation rotation)
     {
+        GlStateManager.translate(0.5F, 0F, 0.5F);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 0.5F to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 0.5F to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)